patches/minecraft/net/minecraft/entity/player/ServerPlayerEntity.java.patch

```diff
@@ -15,15 +15,23 @@
  import javax.annotation.Nullable;
  import net.minecraft.advancements.CriteriaTriggers;
  import net.minecraft.advancements.PlayerAdvancements;
-@@ -464,6 +460,7 @@
+@@ -145,6 +141,7 @@
+    private boolean field_71147_cj = true;
+    private int field_71144_ck = -99999999;
+    private int field_147101_bU = 60;
++   public String language;
```

Comment:
Needs Nullable, and to not be public. We dont need random people trying to set the value and expecting it to do something. And having it a getter method means we can change API easier.